develop-docs/sdk/data-model/envelope-items.mdx

```diff
@@ -427,6 +427,44 @@ details.
 
 *None*
 
+### Logs
+
+Item type `"logs"` contains a logs payload encoded as JSON. Multiple log envelope items can be sent in a single envelope.
+
+See the <Link to="/sdk/telemetry/logs">Logs</Link> documentation for the payload
+details.
+
+**Constraints:**
+
+*None*
+
+**Envelope Headers:**
+
+*None*
+
+**Additional Item Headers:**
+
+*None*
+
+### Otel Logs
+
+Item type `"otel_logs"` contains an OpenTelemetry Logs payload encoded as JSON. Multiple Otel log envelope items can be sent in a single envelope.
+
+See the <Link to="/sdk/telemetry/logs">Logs</Link> documentation for the payload
+details.
+
+**Constraints:**
+
+*None*
+
+**Envelope Headers:**
+
+*None*
+
+**Additional Item Headers:**
+
+*None*
+
```

Comment:
Why do we have both? Our SDKs emit an envelope with the `logs` item type. How does the `otel_logs` envelope item differs here? As in, if we want to support OTLP logs, why not ingesting them as is instead of maintaining two Sentry specific formats?

Reply:
We initially started with `otel_log` as the baseline interface. We wanted to use that as the foundation because it meant at minimum we were OTEL compatible, but it also pointed us to a variety of things to think about.

We are documenting the `otel_log` format for completeness, given we might need to use this in the future.

For the foreseeable future, all SDKs will send events with the `log` payload and envelope item. You can see what this looks like here: https://github.com/getsentry/relay/pull/4592

I've added documentation for the `log` payload, and clarified this: 3ff6e568b5e40648b49c2d64240e45dfb12ae63d